### PR TITLE
Added Euler import connector

### PR DESCRIPTION
**Pull request by shriyatyagii**
Added Euler Import Connector

### CODE REVIEW

**Comment by KABBOUCHI** [comment]
src code is missing
image

File: scripts/connectors.json [hunk 1]
```diff
@@ -215,6 +215,11 @@
       "path": "euler/main.sol",
       "slug": "euler",
       "title": "Euler"
+    },
+    {
+      "path": "euler/euler-import/main.sol",
```
Comment (thrilok209) [inline scripts/connectors.json:220]:
```suggestion
      "path": "euler/import/main.sol",
```

**Comment by thrilok209** [comment]
@KABBOUCHI can you check if everything looks good now?

**Comment by KABBOUCHI** [comment]
> @KABBOUCHI can you check if everything looks good now?

yes its working now https://docs-nuxt-git-euler-import-instadapp-eng.vercel.app/connectors/mainnet/euler-import

**Pull request merged**